settings/IRIS/mclennan/sampleChanger.cmd

```diff
@@ -1,15 +1,16 @@
-$(IFAXIS1) epicsEnvSet "RACKDEFS" "$(ICPCONFIGROOT)/motionSetPoints/rack_definitions.xml"
-$(IFAXIS1) epicsEnvSet "SLOT_DETAILS_FILE" "$(ICPCONFIGROOT)/motionSetPoints/samplechanger.xml"
-$(IFAXIS1) epicsEnvSet "SAMPLE_LKUP_FILE" "$(ICPCONFIGROOT)/motionSetPoints/sample.txt"
+## commented out as local xml/txt files do not seem to exist?
+#$(IFAXIS1) epicsEnvSet "RACKDEFS" "$(ICPCONFIGROOT)/motionSetPoints/rack_definitions.xml"
+#$(IFAXIS1) epicsEnvSet "SLOT_DETAILS_FILE" "$(ICPCONFIGROOT)/motionSetPoints/samplechanger.xml"
+#$(IFAXIS1) epicsEnvSet "SAMPLE_LKUP_FILE" "$(ICPCONFIGROOT)/motionSetPoints/sample.txt"
 
-$(IFAXIS1) sampleChangerConfigure("CURRENT","CURRENT",2)
+#$(IFAXIS1) sampleChangerConfigure("CURRENT","CURRENT",2)
 
-$(IFAXIS1) dbLoadRecords("$(SAMPLECHANGER)/db/sampleChanger.db","P=$(MYPVPREFIX)SAMPCHNG:,TARGET_PV1=$(MYPVPREFIX)LKUP:SAMPLE:RESET,LOOKUP=CURRENT")
+#$(IFAXIS1) dbLoadRecords("$(SAMPLECHANGER)/db/sampleChanger.db","P=$(MYPVPREFIX)SAMPCHNG:,MOT_SET_POINT=$(MYPVPREFIX)LKUP:SAMPLE:,LOOKUP=CURRENT")
```

Comment:
can we just get rid of these lines if not needed? or are they needed?

Reply:
I commented out as the txt/xml didn't exist at the time of testing, but i wasn't sure if they should exist and had somehow got lost. Is/was there another sample changer on IRIS ?